pysilcam/acquisition.py

```diff
@@ -97,30 +106,140 @@ def print_camera_config(camera):
     config_info = '\n'.join(['{0}: {1}'.format(a, camera.getattr(a))
                              for a, b in config_info_map])
 
-    logger.info(config_info) # TODO: Should this be printed?
+    logger.info(config_info)  # TODO: Should this be printed?
     logger.debug(config_info)
 
 
+def silcam_load(filename):
+    #Load the raw image from disc depending on filetype
```

Comment:
E265: block comment should start with '# '  <!-- Automatically posted by Lintly -->